.github/actions/publish-image/action.yml

```diff
@@ -119,7 +119,7 @@ runs:
           --source-version "$IMAGE_MANIFEST_TAG" "${IMAGE_REPO_DIGEST}"
 
         # Merge SBOM components using https://github.com/stackabletech/mergebom
-        curl -L -o mergebom https://repo.stackable.tech/repository/packages/mergebom/stable-$(arch)
+        curl --fail -L -o mergebom https://repo.stackable.tech/repository/packages/mergebom/stable-$(uname -m)
         curl -L -o mergebom_signature.bundle https://repo.stackable.tech/repository/packages/mergebom/stable-$(arch)_signature.bundle
```

Comment:
Still a `curl` here

Reply:
Ah sorry, somehow missed that in the suggested change, fixed